src/logger.ts

```diff
@@ -44,18 +44,9 @@ const severity: Record<keyof AbstractLogger, number> = {
   error: 40,
 };
 
-export const isBuiltinLoggerConfig = (
-  subject: unknown,
-): subject is BuiltinLoggerConfig =>
+export const isActualLogger = (subject: unknown): subject is AbstractLogger =>
   isObject(subject) &&
-  "level" in subject &&
-  ("color" in subject ? typeof subject.color === "boolean" : true) &&
-  ("depth" in subject
-    ? typeof subject.depth === "number" || subject.depth === null
-    : true) &&
-  typeof subject.level === "string" &&
-  ["silent", "warn", "info", "debug"].includes(subject.level) &&
-  !Object.values(subject).some((prop) => typeof prop === "function");
+  Object.keys(severity).some((method) => method in subject);
```

Comment:
> If you set a property to undefined but do not delete it, the in operator returns true for that property.

Reply:
I'm OK with it, because it definitely does not look like a built-in logger config that way